tests/test-inference.js

```diff
@@ -29,6 +29,8 @@ var tests = [
       uniform: { args: { samples: 10000 } },
       beta: true,
       exponential: true,
+      laplace: { args: { samples: 10000 } },
+      tensorLaplace: true,
```

Comment:
Are you OK to add these to the `OptimizeELBO` tests too? (You'll probably have to fiddle around with the inference args to get them passing reliably without having super wide tolerances.)